src/app.routing.ts

```diff
@@ -21,6 +41,6 @@ import { ChartsComponent } from './app/charts/charts.component';
     })
 
     export class AppRouting {}
-    export const Components = [AppComponent,StudentComponent]
+    export const Components = [AppComponent, StudentComponent, NoticePostComponent, NoticeDisplayCompnent, LoginComponent, StudentInfoComponent, StudentListComponent, EventComponent, SchoolGalleryComponent, DashboardComponent]
```

Comment:
line should not be more than 145 characters.

Reply:
Done